Doc/library/xml.etree.elementtree.rst

```diff
@@ -930,6 +938,36 @@ Element Objects
      if element is None:
          print("element not found")
 
+   Prior to Python 3.8, the serialisation order of the attributes of Elements
```

Comment:
Maybe adding a reference to the original class? Feel free to ignore this suggestion :)

```suggestion
Prior to Python 3.8, the serialisation order of the attributes of :class:`Element` objects
```

Reply:
Thanks for the suggestion, but I think this would further increase the ambiguity whether it's referring to the attributes of the Python object or the XML attributes of the XML element. I'll actually change it to "XML attributes" to make that clear.